notebooks/0111_PRIMAP-histTP-prepare.py

```diff
@@ -0,0 +1,98 @@
+# ---
+# jupyter:
+#   jupytext:
+#     text_representation:
+#       extension: .py
+#       format_name: percent
+#       format_version: '1.3'
+#       jupytext_version: 1.16.6
+#   kernelspec:
+#     display_name: Python 3 (ipykernel)
+#     language: python
+#     name: python3
+# ---
+
+# %% [markdown]
+# # Prepare PRIMAP-histTP
+#
+# Prepare data from PRIMAP-histTP.
+
+# %%
+# import external packages and functions
+from pathlib import Path
+
+import numpy as np
+import pandas as pd
+
+from emissions_harmonization_historical.constants import (
+    DATA_ROOT,
+    PRIMAP_HIST_PROCESSING_ID,
+)
+
+# %%
+primap_input_file = DATA_ROOT / Path(
+    "national", "primap-hist", "data_raw", "Guetschow_et_al_2024a-PRIMAP-hist_v2.6_final_no_rounding_13-Sep-2024.csv"
```

Comment:
You'll probably need to run `git add -f data/national/primap-hist/README*` so that the info about how to get this is available (I haven't run it as a result of not having the raw data)